seacrowd/sea_datasets/ccmatrix/ccmatrix.py

```diff
@@ -0,0 +1,269 @@
+    doi = "10.18653/v1/2021.acl-long.507",
+    pages = "6490--6500",
+    abstract = "We show that margin-based bitext mining in a multilingual sentence space can be successfully scaled to operate on monolingual corpora of billions of sentences. We use 32 snapshots of a curated common crawl corpus (Wenzel et al, 2019) totaling 71 billion unique sentences. Using one unified approach for 90 languages, we were able to mine 10.8 billion parallel sentences, out of which only 2.9 billions are aligned with English. We illustrate the capability of our scalable mining system to create high quality training sets from one language to any other by training hundreds of different machine translation models and evaluating them on the many-to-many TED benchmark. Further, we evaluate on competitive translation benchmarks such as WMT and WAT. Using only mined bitext, we set a new state of the art for a single system on the WMT{'}19 test set for English-German/Russian/Chinese. In particular, our English/German and English/Russian systems outperform the best single ones by over 4 BLEU points and are on par with best WMT{'}19 systems, which train on the WMT training data and augment it with backtranslation. We also achieve excellent results for distant languages pairs like Russian/Japanese, outperforming the best submission at the 2020 WAT workshop. All of the mined bitext will be freely available.",
+}
+"""
+
+_DATASETNAME = "ccmatrix"
+
+_DESCRIPTION = """\
+The CCMatrix dataset was collected from web crawls and released by Meta. The dataset is constructed based on the margin-based bitext mining which can be applied to monolingual corpora of billions of sentences to produce high quality aligned translation data.
+"""
+
+_HOMEPAGE = "https://opus.nlpl.eu/CCMatrix/corpus/version/CCMatrix"
+
+_LANGUAGES = ["jav", "eng", "vie", "ind", "tgl", "mya", "zlm"]
+
+_LICENSE = Licenses.BSD.value
+
+_LOCAL = False
+
+_FILE = "CCMatrix.{}.{}"  # E.g. CCMatrix.en-nl.nl
+
+_URLS = "https://object.pouta.csc.fi/OPUS-CCMatrix/v1/moses/{}.txt.zip"
+
+_SUPPORTED_TASKS = [Tasks.MACHINE_TRANSLATION]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class CCMatrixDataset(datasets.GeneratorBasedBuilder):
+    """The CCMatrix dataset was collected from web crawls and released by Meta. The dataset is constructed based on the margin-based bitext mining which can be applied to monolingual corpora of billions of sentences to produce high quality aligned translation data."""
+
+    SEACROWD_SCHEMA = TASK_TO_SCHEMA[Tasks.MACHINE_TRANSLATION].lower()
+
+    LANG_PAIRS = [
+        ("en", "jv"), ("id", "jv"), 
+        ("jv", "tl"), ("jv", "ms"), 
+        ("en", "vi"), ("en", "id"), 
+        ("en", "tl"), ("en", "ms"), 
+        ("id", "vi"), ("tl", "vi"), 
+        ("ms", "vi"), ("id", "tl"), 
+        ("id", "ms"), ("ms", "tl")
+    ]
+
+    BUILDER_CONFIGS = (
+        [
+            SEACrowdConfig(
+                name=f"{_DATASETNAME}_{lang1}-{lang2}_source",
```

Comment:
For the lang code, can we change it so that it follows the same ISO 639-3 standard as in the `_LANGUAGES`?